go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/rogpeppe/modcop
+
+go 1.12
```

Comment:
1.11?

Reply:
This is just what you get if you use go tip to build. Does it matter?

Reply:
This is just what you get if you use go tip to build. Does it matter?